examples/cpp/continuousquery/main.cpp

```diff
@@ -32,65 +33,75 @@ using namespace apache::geode::client;
 using namespace continuousquery;
 
 class MyCqListener : public CqListener {
- public:
-  void onEvent(const CqEvent& cqEvent) {
+public:
+  void onEvent(const CqEvent &cqEvent) override {
```

Comment:
Formatting? Why did you move from right to left spacing? I believe all the other sources and most of Geode is right spaced.

Reply:
I ran the clang-format tool in CLion, apparently it didn't do everything "our" way, go figure.

Reply:
If the file has a majority in one direction the formatting rule puts them all to the majority direction. When starting a new file you should prefer right. I suggest updating your formatting rules in CLion to auto format right and then clang format will keep the file in line.